test/e2e/network_test.go

```diff
@@ -20,9 +20,13 @@ import (
 
 // testNetwork runs all the cluster and node network tests
 func testNetwork(t *testing.T) {
+	testCtx, err := NewTestContext(t)
+	require.NoError(t, err)
+	require.NoError(t, testCtx.createNamespace(testNamespace), "error creating test namespace")
 	t.Run("East West Networking across Linux and Windows nodes", testEastWestNetworking)
 	t.Run("East West Networking across Windows nodes", testEastWestNetworkingAcrossWindowsNodes)
 	t.Run("North south networking", testNorthSouthNetworking)
+	assert.NoError(t, testCtx.deleteNamespace(testNamespace), "unable to delete test namespace")
```

Comment:
Do we need to fail the test because we are not able to cleanup? Cluster tear down will anyhow take care of this.